ramls/types/resources/resourcePutDataAttributes.json

```diff
@@ -110,6 +110,26 @@
         "$ref": "../coverage.json"
       }
     },
+    "userDefinedField1": {
+      "type": "string",
```

Comment:
correct me if im wrong, the fields have length limitation, don't they?
do we want to apply it here (maxLength)?

Reply:
yes, `maxLength` restriction should be added. I thought that custom label value length was covered in other PR. I will update json files for length restriction. Thanks for review @dmtkachenko @carolegodfrey